packages/sources/react/src/components/VtmnTextInput/VtmnTextInput.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+
+import '@vtmn/css-text-input';
+
+export interface VtmnTextInputProps
+  extends React.InputHTMLAttributes<HTMLInputElement> {
+  /**
+   * Label text linked to the input
+   * @type {string}
+   */
+  label: string;
+
+  /**
+   * Helper text to help the user
+   * @type {string}
+   */
+  helperText?: string;
+
+  /**
+   * Whether input is successful or not
+   * @type {boolean}
+   */
+  valid?: boolean;
+
+  /**
+   * Whether input is in error or not
+   * @type {boolean}
+   */
+  error?: boolean;
+}
+
+export const VtmnTextInput = ({
+  className,
+  disabled = false,
+  error = false,
+  helperText,
+  id,
+  label,
+  placeholder,
+  valid = false,
+  ...props
+}: VtmnTextInputProps) => {
+  return (
+    <>
+      <label className="vtmn-text-input_label" htmlFor={id}>
+        {label}
+        <input
+          className={`vtmn-text-input ${valid && 'vtmn-text-input--valid'}
+          ${error && 'vtmn-text-input--error'} ${className ? className : ''}`}
```

Comment:
On pourrait utiliser la lib clsx, qui s'utiliserait de la manière suivante : 
```typescript
const classNames = clsx('vtmn-text-input', className, { 'vtmn-text-input--valid': valid }, { 'vtmn-text-input--error': error });

return (
   <input className={classNames} />
);
```
Histoire d'éviter toutes ces conditions

Reply:
Indeed thanks @samedro for you feedback :)